daffodil-tdml-processor/src/main/scala/org/apache/daffodil/processor/tdml/DaffodilTDMLDFDLProcessor.scala

```diff
@@ -190,8 +190,8 @@ class DaffodilTDMLDFDLProcessor private (private var dp: DataProcessor)
     }
 
   override def withDebugger(db: AnyRef): DaffodilTDMLDFDLProcessor = {
-    Assert.usage(dp ne null)
-    val d = dp.asInstanceOf[Debugger]
+    Assert.usage(db ne null)
+    val d = db.asInstanceOf[Debugger]
```

Comment:
Wow. What a bug. 

We should disallow use of tiny value names like dp and db and spell things out a little more I think. 

db here is a parameter name. That one should definitely be spelled out as 'debugger'.

Reply:
Note that code cov says this wasn't a bug, because it's _not called_.

Reply:
Right, I noticed the bug while working on getting the -d functionality to work with "test", it gets called then. This changeset doesn't have that functionality work, but it felt wrong to leave the bug